git-bz

```diff
@@ -819,7 +834,8 @@ class Bug(object):
             if a["is_obsolete"]:
                 continue
             if a["is_patch"] or a["content_type"] == u'text/x-review-board-request':
-                self.patches.append(BugAttachment(a))
+                creator = self.server.get_user_detail(a['creator'])
```

Comment:
Can this get moved into the |diff.startswith('diff ')| case in BugAttachment? Then we'd skip the BZAPI call unless we really need it.